app/feed_bot.py

```diff
@@ -49,7 +49,7 @@ def main():
                 "title": entry.title,
                 "config": feed,
                 "date": published_date,
-                "rel_file_path": f"{folder}/{file_name}.md",
```

Comment:
why is the md not needed anymore?

Reply:
It will be added in the util.py as we need to create files for separated social media platforms.